hazelcast/src/main/java/com/hazelcast/config/MetricsManagementCenterConfig.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.config;
+
+import com.hazelcast.spi.properties.GroupProperty;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Management Center related metrics configuration.
+ *
+ * @since 4.0
+ */
+public class MetricsManagementCenterConfig {
+
+    /**
+     * Default retention period for metrics.
+     */
+    public static final int DEFAULT_METRICS_RETENTION_SECONDS = 5;
+
+    private boolean enabled = true;
+    private int retentionSeconds = DEFAULT_METRICS_RETENTION_SECONDS;
+
+    public MetricsManagementCenterConfig() {
+    }
+
+    public MetricsManagementCenterConfig(MetricsManagementCenterConfig managementCenterConfig) {
+        this.enabled = managementCenterConfig.enabled;
+        this.retentionSeconds = managementCenterConfig.retentionSeconds;
+    }
+
+    /**
+     * Returns whether metrics will be exposed to Hazelcast Management
+     * Center. If enabled, Hazelcast Management Center will be able to read
+     * out the collected metrics from this member. It's enabled by default.
+     * <p/>
+     * This configuration acts as a fine-tuning option beyond
+     * enabling/disabling the Metrics collection entirely via the
+     * {@link MetricsConfig#isEnabled()} master switch.
+     *
+     * @return true if exposing to Hazelcast Management Center is enabled.
+     * @see MetricsConfig#isEnabled()
+     */
+    public boolean isEnabled() {
+        return enabled;
+    }
+
+    /**
+     * Enables exposing metrics to Hazelcast Management Center. If enabled,
+     * Hazelcast Management Center will be able to read out the recorded
+     * metrics from this member. It's enabled by default.
+     * <p/>
+     * This configuration acts as a fine-tuning option beyond
+     * enabling/disabling the Metrics collection entirely via the {@link #enabled}
```

Comment:
`{@link #enabled}` should be `{@link MetricsConfig#isEnabled()}` here, right? If yes, then the same comment applies to javadocs in `MetricsJmxConfig`.

Reply:
Right, changed.